in-skill-purchase/response/prompts/AlexaConversationsBye/document.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
We don't need any of these default prompts in skill components, only prompts used in the ACDL will actually ever be used